sdk/python/foundation-models/system/import/import_model_into_registry.ipynb

```diff
@@ -272,11 +265,15 @@
     "Browse models on [HuggingFace hub](https://huggingface.co/models) and identify a model to import. Make sure the task type of the model is among the supported tasks as explained in the introduction section. Copy the model id which is available in the URI of the page or can be copied using the copy icon next to the model name and assign it to the variable `MODEL_ID`.\n",
     "\n",
     "\n",
-    "![final-image.png](attachment:final-image.png)"
+    "![final-image.png](attachment:final-image.png)\n",
+    "\n",
+    "\n",
+    "#### token\n",
+    "\n",
+    "There are few gated models hosted in HuggingFace for example [tiiuae/falcon-180B](https://huggingface.co/tiiuae/falcon-180B), which requires user to accept certain terms and condition, and can't be accessed directly to import. To do that user have to create a *personal account* in HuggingFace and accept terms and conditon. To import such models user have to create Access Token in HuggingFace account and pass it to model-import component, so that model can be downloaded in their workspace or registry. To know the more details , please visit [this link](https://huggingface.co/docs/hub/security-tokens)."
```

Comment:
There are few gated models hosted in HuggingFace for example [tiiuae/falcon-180B](https://huggingface.co/tiiuae/falcon-180B), which requires the user to accept certain terms and conditions, and can't be accessed directly to the import component. To do so, the user would have to create a *personal account* in HuggingFace and accept terms and conditions. To import such models the user would have to create an Access Token with their HuggingFace account and pass it to the model-import component, after which the model can be downloaded to their workspace or registry. To know the more details , please visit [this link](https://huggingface.co/docs/hub/security-tokens).